src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -82,14 +65,20 @@ def __init__(self, ensemble: Ensemble, config: EvaluatorServerConfig):
         self._max_batch_size: int = 500
         self._batching_interval: float = 2.0
         self._complete_batch: asyncio.Event = asyncio.Event()
+        self._server_started: asyncio.Event = asyncio.Event()
+        self._clients_connected: set[bytes] = set()
+        self._clients_empty: asyncio.Event = asyncio.Event()
+        self._dispatchers_connected: set[bytes] = set()
+        self._dispatchers_empty: asyncio.Event = asyncio.Event()
 
     async def _publisher(self) -> None:
+        await self._server_started.wait()
```

Comment:
Do we need this here?

Reply:
I think this one is mainly for the tests, which not always run the full evaluation.